src/game.cpp

```diff
@@ -5742,6 +5758,13 @@ int game::assign_faction_id()
     return ret;
 }
 
+long game::assign_creature_id()
+{
+    long ret = next_creature_id;
```

Comment:
I wanted to go with `unsigned long` at first, but json.h doesn't seem to support it. Should I add reading and writing those to json.h?